sdk/ml/azure-ai-ml/azure/ai/ml/entities/_inputs_outputs/input.py

```diff
@@ -40,6 +40,8 @@ class Input(_InputOutputBase):  # pylint: disable=too-many-instance-attributes
         * 'download': Download the data to the compute target,
         * 'direct': Pass in the URI as a string to be accessed at runtime
     :paramtype mode: Optional[str]
+    :keyword path_on_compute: The access path of the data input for compute
```

Comment:
pls add some test for this change

Reply:
Since the Joboutput and JobInput model of current version does not support path_on_compute yet, I can not test functions such as from_rest_data_outputs etc. I plan to add more tests when the model enables new JobInput and JobOutput